config/auth_actions.php.default

```diff
@@ -20,5 +20,8 @@ $config = [
 		'my_account' => [
 			'*' => '*'
 		]
+	],
+	'auto_login' => [
+	    'salt' =>
```

Comment:
Unexpected `]`, expected `'` or any value, anything really, but not closing the array. Please?